src/mgmt/srv_pool.c

```diff
@@ -49,7 +49,7 @@ ds_mgmt_tgt_pool_destroy(uuid_t pool_uuid, crt_group_t *grp)
 	topo = crt_tree_topo(CRT_TREE_KNOMIAL, 4);
 	opc = DAOS_RPC_OPCODE(MGMT_TGT_DESTROY, DAOS_MGMT_MODULE,
 			      DAOS_MGMT_VERSION);
-	rc = crt_corpc_req_create(dss_get_module_info()->dmi_ctx, grp,
+	rc = crt_corpc_req_create(dss_get_module_info()->dmi_ctx, NULL,
```

Comment:
so cart api will replace this grp with something like rank_list?

Reply:
This code path is unchanged: still primary group + excluded ranks. Here I just discover we always use PG, and do not need an input grp parameter.